src/EditorFeatures/CSharpTest/RemoveUnusedVariable/RemoveUnusedVariableTests.cs

```diff
@@ -228,42 +228,36 @@ public async Task LeadingDirectives()
 
 using System;
 
-namespace ClassLibrary
```

Comment:
please don't edit test inputs. a dd new tests.

Reply:
I suspect this is a moot change now anyway, but I don't think I understand the idea behind this. If the logic in the fixer changes, such that a test is now failing (in this case because namespace declarations aren't allowed in script mode), why is it a bad idea to change the test so that it passes. As long as the change doesn't make the test irrelevant, of course. Or do you mean you'd rather remove and re-add it somewhere else in the file so the git diff is clearer?

Reply:
> why is it a bad idea to change the test so that it passes. 

because that input still represents a valuable case that exercises the codepath.  If we think we have a different input we want to test with, we can add new tests.  but this sort of change effectively removes an existing test entirely, which means we might miss something.